ksql-rest-app/src/main/java/io/confluent/ksql/api/server/ApiServerConfig.java

```diff
@@ -38,18 +38,13 @@
   public static final int DEFAULT_VERTICLE_INSTANCES =
       2 * Runtime.getRuntime().availableProcessors();
   public static final String VERTICLE_INSTANCES_DOC =
-      "The number of server verticle instances to start. Usually you want at least many instances"
-          + " as there are cores you want to use, as each instance is single threaded.";
+      "The number of server verticle instances to start per listener. Usually you want at least "
+          + "many instances as there are cores you want to use, as each instance is single "
+          + "threaded.";
 
-  public static final String LISTEN_HOST = propertyName("listen.host");
-  public static final String DEFAULT_LISTEN_HOST = "localhost";
-  public static final String LISTEN_HOST_DOC =
-      "The hostname to listen on";
-
-  public static final String LISTEN_PORT = propertyName("listen.port");
-  public static final int DEFAULT_LISTEN_PORT = 8088;
-  public static final String LISTEN_PORT_DOC =
-      "The port to listen on";
+  public static final String LISTENERS = propertyName("listeners");
+  public static final String DEFAULT_LISTENERS = "http://0.0.0.0:8088";
+  public static final String LISTENERS_DOC = "List of listeners to listen for connections on";
 
   public static final String TLS_ENABLED = propertyName("tls.enabled");
```

Comment:
This property is unused now and may be removed, right?

Reply:
ack